defaults/main.yml

```diff
@@ -79,10 +79,10 @@ enketo_maps:
   attribution: "Map data © <a href='http://openstreetmap.org'>OpenStreetMap</a> contributors"
 enketo_google_analytics_ua: ""
 enketo_google_analytics_domain: ""
-enketo_google_api_key: "{{ enketo_google_api_key|default('') }}"
+enketo_google_api_key: ""
```

Comment:
Why did we replace `{{ enketo_google_api_key|default('') }} ` value with ""

Reply:
This caused an infinite loop when `enketo_google_api_key` is not set.

Reply:
Hmm....that's interesting. Was molecule the one complaining? I'm asking because the above directs ansible to use `''` only when `enketo_google_api_key` is not set and that seems like valid ansible syntax, no? https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#id3

Reply:
Also, is this PR introducing molecule or that already existed?

Reply:
This PR does introduce molecule

Yes, molecule threw an error. With this change from `"{{ enketo_google_api_key|default('') }}"` to `""` in the value of `enketo_google_api_key` we still achieve intended behaviour; when you don't set `enketo_google_api_key` in your inventories, it's set to `""`. when you do set it in your inventories, it will set to the value you have set in the inventories